ratpack-kafka-producer/src/main/java/smartthings/ratpack/kafka/KafkaProducerModule.java

```diff
@@ -26,6 +27,12 @@ protected void configure() {
 		String clientId;
 		Long maxBlockMillis = TimeUnit.MINUTES.toMillis(1);
 		boolean enabled = true;
+		Optional<Long> lingers_ms;
```

Comment:
You'll still need to set these properties in for the changes to make it to the producer.  I was just saying only set them if the user actually specifies a property, and don't set anything if they don't.